doc/sample_azure-keyvault.rst

```diff
@@ -22,7 +22,24 @@ for details on handling Azure Active Directory authentication with the Python SD
     credentials = UserPassCredentials(
         'user@domain.com',  # Your user
         'my_password',      # Your password
-        resource="https://vault.azure.net"
+        resource='https://vault.azure.net'
+    )
+
+    client = KeyVaultClient(
+        credentials
+    )
+
+.. code:: python
```

Comment:
I added this because I would think this use case, accessing keys and secrets, is just as likely from a service context than under a user context. I don't see a problem removing it from this PR if its addition creates a burden for the maintenance of the documentation.